content/en/docs/Configuration/multi-cluster.md

```diff
@@ -11,11 +11,9 @@ Before proceeding with the setup, ensure you meet the requirements.
 
 ### Requirements
 
-1. **Primary-remote istio deployment.** Only the primary-remote istio deployment is currently supported.
+1. **Aggregated metrics and traces.** Kiali needs a single endpoint for metrics and a single endpoint for traces where it can consume aggregated metrics/traces across all clusters. There are many ways to aggregate metrics/traces such as prometheus federation or using OTEL collector pipelines but setting these up are outside of the scope of Kiali.
```

Comment:
nit.. but I think we should capitalize Prometheus always
```suggestion
1. **Aggregated metrics and traces.** Kiali needs a single endpoint for metrics and a single endpoint for traces where it can consume aggregated metrics/traces across all clusters. There are many ways to aggregate metrics/traces such as Prometheus federation or using OTEL collector pipelines but setting these up are outside of the scope of Kiali.
```